source/common/websocket/codec.cc

```diff
@@ -86,7 +83,8 @@ uint8_t Decoder::doDecodeFlagsAndOpcode(absl::Span<const uint8_t>& data) {
   // Validate opcode (last 4 bits)
   uint8_t opcode = data.front() & 0x0f;
   if (std::find(kFrameOpcodes.begin(), kFrameOpcodes.end(), opcode) == kFrameOpcodes.end()) {
-    return false;
+    ENVOY_LOG(error, "Failed to decode websocket frame with invalid opcode: {}", opcode);
```

Comment:
Hm, I'm not sure about this being an error. Since this input will be coming from the peer (who is potentially an attacker) this means that an attacker could trigger this message repeatedly. As I understand it, logging can be quite expensive so this might be a DoS risk. How about debug?